zbus/src/xml.rs

```diff
@@ -183,7 +185,7 @@ pub struct Property<'a> {
     #[serde(rename = "@name", borrow)]
     name: MemberName<'a>,
 
-    #[serde(rename = "@type")]
+    #[serde(deserialize_with = "de_type_valid", rename = "@type")]
```

Comment:
> We could have used zv:Signature instead of String here, I guess.

Why don't we switch now? Remember, we're in API break and this seems like a good candidate for breakage.

Reply:
done, with the introduction of CompleteType (which I think should be used in many places for correctness)

Reply:
You mean you change this code to use the new type in a later commit? Why not move the commit that introduces this type before this one and update this commit to use it already? It would have made review easier too.